### DISCUSSION TITLE
Mau minta saran puh

### DISCUSSION THREAD

**Discussion by zero-termux** [deaafrizal/forum#59] [The Lounge]
Gw rencana mau belajar react tpi gw lihat di yt pak dika katanya harus belajar node.js dlu  saran nya gw lanjutin react dlu apa node.js

**Comment by danielpsrb** [comment]
react dan node js itu beda ya meskipun masih dalam aliran javascript

kalau react itu library untuk buat ui javascript ini sebagai front end, nah kalau node js itu lebih fokus ke backend untuk pembuatan server dan API Database ya

saran saya pelajari react dlu, ntr kalau udh menguasai baru masuk node js

**Reply by yohanesokta** [reply]
Nice bang , tapi running react tetep butuh node js , mungkin bisa belajar install install package dulu 😁

**Comment by rexyb9** [comment]
mending belajar dulu dasarnya
